adds batch 5 of bugs to RNs (#48433)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

* Previously, the procedural name generator for Azure availability sets exceeded the 80 character maximum limit. This might cause the Machine API to reuse the same sets during the name truncation, rather than creating multiple availability sets. With this release, the procedural name generator is updated to ensure that the name is no longer than 80 characters and the cluster name is not duplicated in the set name. As a result, Azure availability sets are no longer truncated in unexpected ways by the procedural name generator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2093044[*BZ#2093044*])

> * Because the Cluster Autoscaler Operator was deploying the cluster autoscaler without setting any leader election parameters, the cluster autoscaler could unexpectedly fail and restart after a cluster restart. With this fix, the Cluster Autoscaler Operator now deploys the cluster autoscaler with well-defined leader election flags. As a result, the cluster autoscaler operates as expected after restarts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2063194[*BZ#2063194*])"

Reviewer: When referring to the (Cluster Autoscaler) Operator, initial caps are used and when not talking about the Operator, it is all lowercase...just want to confirm?

Author: I just did a search, looks like maybe lowercase?? so cluster autoscaler Operator. Going to wait for confirmation.

Reviewer 2: "cluster autoscaler" and "Cluster Autoscaler Operator"

- https://docs.openshift.com/container-platform/4.10/machine_management/applying-autoscaling.html
- https://docs.openshift.com/container-platform/4.10/operators/operator-reference.html#cluster-autoscaler-operator_platform-operators-ref

[id="ocp-4-11-image-registry-bug-fixes"]
==== Image Registry

> * Previously, the Image Registry Operator is degraded. This can happen if the pruner fails. The operator remains degraded until the pruner successfully runs. By default, the pruner runs once a day, but can be run more often. When a successful run is completed, the Image Registry Operator is no longer degraded. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990125[*BZ#1990125*])

Reviewer 2: This seems more like a known issue than a bug fix:

```suggestion
* If the pruner fails, the Image Registry Operator is degraded until the pruner successfully runs. By default, the pruner runs once a day. As a workaround, you can give it more opportunities to run successfully by configuring it to run more often. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990125[*BZ#1990125*])
```

Ideally would want to tell them _how_ to make the pruner run more frequently.

Author: Thats what I was thinking too.... It felt kinda weird to put it in the bug fix section.

> * Previously, the image registry used ICSPs as a source only when it was an exact match. The same source file was expected to be pulled through for any sub-repositories.The ICSP name/paths did not match for sub-repositories. As a result, the image is not used. Now the ICSP is applied successfully to the sub-repositories. which can use the mirrored images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014240[*BZ#2014240*])

Reviewer 2: Should define ICSP on first appearance

```suggestion
* Previously, the image registry used an ICSP as a source only when it was an exact match. The same source file was expected to be pulled through for any subrepositories. The ICSP name and path did not match for subrepositories. As a result, the image was not used. Now, the ICSP is applied successfully to the subrepositories, which can use the mirrored images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014240[*BZ#2014240*])
```

[id="ocp-4-11-openshift-api-server-bug-fixes"]
==== OpenShift API server

* Because multiple Authentication Operator controllers were synchronizing at the same time, the Authentication Operator was taking too long to react to changes to its configuration. This fix adds jitter to the regular synchronization periods so that the Authentication Operator controllers do not race for resources. As a result, it now takes less time for the Authentication Operator to react to configuration changes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1958198[*BZ#1958198*])"

> * With {product-title} 4.11, authentication attempts from external identity providers are now logged to the audit logs. As a result, you can view successful, failed, and errored login attempts from external identity providers in the audit logs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2086465[*BZ#2086465*])"

Reviewer 2: Feature rather than fix?

* Previously, nodes that were provisioned from VMware OVA images did not delete the Ignition config after initial provisioning. Consequently, this created security issues when secrets are stored within the Ignition config. With this update, the Ignition config is now deleted from the VMware hypervisor after initial provisioning on new nodes and when upgrading from a previous {product-title} release on existing nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082274[BZ#2082274])

> * Previously, any arguments provided to the `toolbox` command were ignored when the command is first invoked. This fix updates the toolbox script to initiate `podman container create` followed by `podman start` and `podman exec` as well as modifies the script to handle multiple arguments and white spaces as an array. As a result, the arguments passed to the `toolbox` command are executed every time as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039589[*BZ#2039589*])

Reviewer: ```suggestion
* Previously, any arguments provided to the `toolbox` command were ignored when the command is first invoked. This fix updates the toolbox script to initiate `podman container create` followed by `podman start` and `podman exec`, as well as modifies the script to handle multiple arguments and white spaces as an array. As a result, the arguments passed to the `toolbox` command are executed every time as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039589[*BZ#2039589*])
```

> * Previously, any arguments provided to the `toolbox` command were ignored when the command is first invoked. This fix updates the toolbox script to initiate `podman container create` followed by `podman start` and `podman exec` as well as modifies the script to handle multiple arguments and white spaces as an array. As a result, the arguments passed to the `toolbox` command are executed every time as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039589[*BZ#2039589*])

Reviewer 2: ```suggestion
* Previously, any arguments provided to the `toolbox` command were ignored when the command was first invoked. This fix updates the toolbox script to initiate the `podman container create` command followed by the `podman start` and `podman exec` commands. It also modifies the script to handle multiple arguments and whitespaces as an array. As a result, the arguments passed to the `toolbox` command are executed every time as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039589[*BZ#2039589*])
```
